src/main/java/uk/gov/digital/ho/hocs/casework/client/infoclient/InfoClient.java

```diff
@@ -30,24 +30,9 @@ public InfoClient(RestHelper restHelper,
         this.serviceBaseURL = infoService;
     }
 
```

Comment:
We can reduce this to one call that fetches all caseDataTypes - we can present it differently in the service layer. it also allows us to deprecate or remove these calls from info-service.